msgraph/models.go

```diff
@@ -646,6 +656,7 @@ type ConditionalAccessFilter struct {
 
 type ConditionalAccessGrantControls struct {
 	Operator                    *string                          `json:"operator,omitempty"`
+	AuthenticationStrength      *[]AuthenticationStrengthPolicy  `json:"authenticationStrength,omitempty"`
```

Comment:
It looks like this only accepts a single authenticationStrengthPolicy per conditionalAccessPolicy
```suggestion
	AuthenticationStrength      *AuthenticationStrengthPolicy  `json:"authenticationStrength,omitempty"`
```